assets/section-footer.css

```diff
@@ -121,20 +121,15 @@
 
 .footer-block--newsletter {
   display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
   align-items: flex-end;
   margin-top: 3rem;
+  gap: 1rem;
```

Comment:
Here I'm wondering if we should use the variables we already have instead of a hard coded value. The blocks coming up in the footer are using `column-gap: var(--grid-desktop-horizontal-spacing);` and `column-gap: var(--grid-mobile-horizontal-spacing);`.

Not a blocker, just a thought so everything is using a similar spacing.

Reply:
Oh maybe it's due to the `flex-direction` on mobile ? Since it's column 🤔